frontend/src/utils.js

```diff
@@ -8,11 +8,10 @@ export function pluralize(singular, plural, count) {
      *
      * @return  {String}            Proper form of the word
      **/
-    if (count > 1) {
-        return plural
-    } else {
+    if (count == 1) {
```

Comment:
Are you sure about this? My research showed it's "0.5 inch" rather than "0.5 inches"?

Also, thing to remember: float equality is dangerous because 0.0 === 0.0 is not guaranteed because the numbers are not necessarily exact. That's why people usually replace equality evaluation with checking that the difference is smaller than a tiny threshold (e.g. 1e-5). If you are interested you can look into how floating point representation works.

Reply:
Not sure since I didn’t do any research. But “0.5 inches” (“zero point five”) sounded more natural to me than “0.5 inch” unless you’re saying it like “half of an inch”. Maybe we can ask at standup tomorrow what the others think.

Thank you for the reminder about floating equality. I had forgotten about it.